x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/add_role_mapping_button.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButtonTo } from '../react_router_helpers';
+
+import { ADD_ROLE_MAPPING_BUTTON } from './constants';
+
+interface IAddRoleMappingButtonProps {
```

Comment:
I think we've moved away from prefixes on types. We just call local prop interfaces that are not exported `Props` for most of our AS components. `AddRoleMappingButtonProps` would work as well.